saas/fsm/unsanctioned_spend/README.md

```diff
@@ -0,0 +1,29 @@
+# SaaS Manager - Unsanctioned Spend Alert
+
+## What it does
+
+This policy will raise an alert when Flexera SaaS Manager identifies unsanctioned spend on SaaS applications.
```

Comment:
product doesn't want to use alert.  instead change **raise an alert** to **create an incident**.